.github/workflows/ci.yml

```diff
@@ -445,3 +445,14 @@ jobs:
           # comes from the get-secrets step (shared get-vault-secrets workflow)
           SLACK_WEBHOOK_URL: ${{ env.SLACK_WEBHOOK_URL }}
         run: npm run release
+
+      - name: Trigger examples update
+        run: |
```

Comment:
Is it much of an issue if this triggers examples repo CI on every merge to main?

Reply:
Good point. I should add a check to only if files inside create-plugin package were touched